src/aap_eda/api/serializers/activation.py

```diff
@@ -411,7 +411,10 @@ class Meta:
         required=False,
         allow_null=True,
         child=serializers.IntegerField(),
-        validators=[validators.check_multiple_credentials],
+        validators=[
+            validators.check_multiple_credentials,
+            validators.check_if_has_multiple_aap_credentials,
```

Comment:
@hsong-rh Would a better name for this function be

* check_single_aap_credential

Reply:
Changed